src/borg/helpers/parseformat.py

```diff
@@ -764,7 +764,7 @@ def __init__(self, archive, format, *, json_lines=False):
             'atime': partial(self.format_time, 'atime'),
         }
         for hash_function in self.hash_algorithms:
-            self.add_key(hash_function, partial(self.hash_item, hash_function))
+            self.call_keys[hash_function] = partial(self.hash_item, hash_function)
```

Comment:
does this remove the only usage of `add_key`?

if so, one could either remove that also **or** accept the little overhead and **only** use add_key always.